src/components/SearchAndFilter/FilterPanelSection/FilterPanelSection.js

```diff
@@ -7,12 +7,13 @@ import "./filter-panel-section.scss";
 const FilterPanelSection = ({ data }) => {
   const { chips, heading } = data;
   const [overflowCounter, setOverflowCounter] = useState(0);
+  const [expanded, setExpanded] = useState(false);
   const chipWrapper = useRef(null);
 
   // If the offsetTop is more than double height of a single chip, consider it
   // overflowing
   const updateFlowCount = function () {
-    const chips = chipWrapper?.current.querySelectorAll(".p-chip");
+    const chips = chipWrapper?.current?.querySelectorAll(".p-chip");
     let overflowChips = 0;
     chips.forEach((chip) => {
```

Comment:
Because of the change on 16 there you'll want to add a check that `chips` exists. If 16 short circuits then you'll end up with `undefined.forEach` which will fail.

Reply:
Good catch, thanks